.github/workflows/build_runner_image.yml

```diff
@@ -19,9 +19,14 @@
 name: Build and Version Runner Docker Image
 
 on:
+  push:
+    tags: ['v*']
+    branches: ['master', 'release-*']
```

Comment:
This probably only needs to run on pushes to master, right? Will we ever use the release-* or v* tags?

Reply:
Correct. Image is not part of the release cycle. Copied over and will remove it